first-deliverable/main.go

```diff
@@ -0,0 +1,96 @@
+package main
+
+import (
+	"encoding/csv"
+	"errors"
+	"fmt"
+	"github.com/go-chi/chi/v5"
+	"github.com/go-chi/chi/v5/middleware"
+	"github.com/go-chi/render"
+	"log"
+	"net/http"
+	"os"
+)
+
+type IdRecord struct {
```

Comment:
This may not be required to be imported as it is in the same package as all the other functions, a good practice is that every element (func, struct, method, interface) it is recommended to left a comment.

// IdRecord - This is a structure that ... it contains... 
// this could also be multi line if required.
type IdRecord struct